cluster-operator/src/main/java/io/strimzi/operator/cluster/model/cruisecontrol/CapacityConfiguration.java

```diff
@@ -208,9 +210,30 @@ private static TreeMap<Integer, CapacityEntry> generateCapacityEntries(Reconcili
             capacityEntries.put(node.nodeId(), new CapacityEntry(node.nodeId(), disk, cpu, inboundNetwork, outboundNetwork));
         }
 
+        isInboundNetworkConfigured = InboundNetworkCapacity.isCapacityConfigured(commonBrokerCapacity);
+        isOutboundNetworkConfigured = OutboundNetworkCapacity.isCapacityConfigured(commonBrokerCapacity);
```

Comment:
Any reason why these go through static methods? Why isn't it directly part of the objects directly? Also, why does it not care about the overrides?

Reply:
Fixed.